decentraland/unity-explorer
# style: few UI updates (mini games map pin, notifications indicator, badge progress bar)

## What does this PR change?

This PR was created to enhance mini map pins resolution, to align the notifications indicator with the design criteria, and to fix badges progress bar texture settings which gradient was looking broken.

## How to test the changes?

1. Launch the explorer
2. Write in the chat input `/loadpx globalpx`. Once the scene loaded and the popup appears, open de Nav Map and check the mini-games pins resolution is enhanced.

<img width="483" alt="Screenshot 2024-09-30 at 11 33 49" src="https://github.com/user-attachments/assets/d51b46b3-d37f-44d6-b096-d780cd997710">

3. Open your profile and go to the badges section. Check the progress bar gradient between colours ins now smooth instead of chopped between orange and yellow.

<img width="752" alt="Screenshot 2024-09-30 at 11 34 47" src="https://github.com/user-attachments/assets/48bf3c46-6f19-4d72-ba0e-0412f20d4651">

4. In case you have unread notifications check the indicator matches the design. I don't know which is the hack to receive a new notification but the update is there XD

<img width="223" alt="Screenshot 2024-09-30 at 11 33 16" src="https://github.com/user-attachments/assets/da32d792-d662-44a6-aaf9-5f89dbfd92cd">
